packages/beapi/src/command/executeCommand.ts

```diff
@@ -1,31 +1,69 @@
-import { world } from 'mojang-minecraft'
-import type { ExecuteCommandResponse, Dimensions } from '../@types/BeAPI.i'
+import { world } from "mojang-minecraft";
+import type { ExecuteCommandResponse, Dimensions } from "../@types/BeAPI.i";
 
-function executeCommand(cmd: string, dimension?: Dimensions, debug = false): ExecuteCommandResponse {
-  const command = cmd.replace(/\\/g, '')
+function executeCommand(
+  cmd: string,
+  dimension?: Dimensions,
+  debug = false
+): ExecuteCommandResponse {
+  const command = cmd.replace(/\\/g, "");
 
   try {
-    if (!dimension) dimension = 'overworld'
-    const cmd = world.getDimension(dimension).runCommand(command)
+    if (!dimension) dimension = "overworld";
+    const cmd = world.getDimension(dimension).runCommand(command);
 
     return {
       statusMessage: cmd.statusMessage,
       data: cmd,
       err: false,
-    }
+    };
   } catch (err) {
     if (!debug)
       return {
         statusMessage: `Error Occured: ${err}`,
         err: true,
-      }
-    console.warn(`[BeAPI] [executeCommand]${err}`)
+      };
+    console.warn(`[BeAPI] [executeCommand]${err}`);
 
     return {
-      statusMessage: 'Error Occured',
+      statusMessage: "Error Occured",
       err: true,
-    }
+    };
   }
 }
 
-export { executeCommand }
+function executeCommands(
+  cmds: Array<string>,
+  dimension?: Dimensions,
+  debug = false
+): ExecuteCommandResponse {
+
+  try {
+    if (!dimension) dimension = "overworld";
```

Comment:
Rather than checking if dimension is provided make it the variable default. 
EG:
```js
function myMethod(paramater = 'default'): void {
  console.log(parameter)
}
```